src/app/app.module.ts

```diff
@@ -23,7 +39,9 @@ import { ErrorTailorModule } from '@ngneat/error-tailor';
           };
         },
         deps: []
-      }
+      },
+      controlErrorComponent: CustomControlErrorComponent, // Uncomment to see errors being rendered using a custom component
```

Comment:
Uncomment again :)

Reply:
oops

Reply:
Shall I remove the whole custom control specification from the sample app? Sample code for this in the README and I believe is well captured there. So may be it's not necessary?

Reply:
No, just put it in a comment. It's good for testing.

Reply:
Ok